creator-node/src/ImageProcessingQueue.ts

```diff
@@ -1,19 +1,19 @@
-const { Queue, QueueEvents, Worker } = require('bullmq')
-const path = require('path')
-const os = require('os')
+import type { PrometheusRegistry } from './services/prometheusMonitoring/prometheusRegistry'
+import { Queue, QueueEvents, Worker } from 'bullmq'
+import path from 'path'
+import os from 'os'
```

Comment:
could you double-check that MAX_CONCURRENCY is accurate when using os.cpus().length? you could just add a console log for os.cpus().length. I'm saying this because I remember having issues when using import syntax with os, but I'm not sure if it was an immediate issue with compiling or if it was a sneaky one where the functions on `os` just don't work. other people had some related issues as well - eg https://stackoverflow.com/questions/72245507/why-i-cant-import-os-modules-with-import-syntax

Reply:
I tested it locally and it seemed to work

Reply:
It returned the correct number for my dev box and laptop

Reply:
awesome